README.md

```diff
@@ -62,13 +62,28 @@ To test Helm Charts locally you will need to follow the next steps:
 13. Confirm it's gone: `helm list && kubectl get pods`
 14. Shutdown minikube: `minikube stop`
 
-To run unit tests:
+### Running Unit Tests
+To unit test the helm charts tou can follow the next steps
```

Comment:
typo: "tou"

Reply:
Thanks for catching this John, I am going to fix it.